minifi/minifi-bootstrap/src/test/java/org/apache/nifi/minifi/bootstrap/RunMiNiFiTest.java

```diff
@@ -73,7 +71,7 @@ public void buildSecurityPropertiesDefined() throws Exception {
 
     @Test
     public void buildSecurityPropertiesDefinedButInvalid() throws Exception {
-        final RunMiNiFi testMiNiFi = new RunMiNiFi(null);
+        //final RunMiNiFi testMiNiFi = new RunMiNiFi(null);
```

Comment:
Not needed?

Reply:
Seems this whole RunMiNiFiTest verify only the Bootstrap file transformer and not the runner. (and caused NPE with the changes (: ) I'm removing the comment.